src/index.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
+import App from './App/App.js';
```

Comment:
Glad we got these files organized a bit. It was getting a little busy in our src folder